components/InteractionDetailsHeader/InteractionDetailsHeader.styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const StyledHeader = styled.div`
+  display: flex;
+`;
+
+export const DetailsContainer = styled.div`
+  width: 100%;
+  margin: 10px;
+`;
+
+export const InteractionTypeContainer = styled.h1`
+  font-size: 1.3rem;
+`;
+
+export const DateContainer = styled.div`
+  margin-top: 5px;
+  font-size: 0.8rem;
+`;
```

Comment:
I recommend that you review the choice of HTML elements and the naming of your components. For example, the name `InteractionTypeContainer` for an `h1 element` is misleading.
I also wonder why the `StyledHeader` component is a `div `element and not a `header` element. 🤔